CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Following Semantic Versioning 2.
 
 ## next version:
 
+## Version 0.0.16 (MINOR)
```

Comment:
it is like a major but as we're still in beta, we use MINOR version indeed.
But then the new version should be `0.1.0`.